src/DIRAC/FrameworkSystem/Client/UserProfileClient.py

```diff
@@ -17,7 +17,7 @@ def __getRPCClient(self):
 
     def storeVar(self, varName, data, perms={}):
         try:
-            stub = DEncode.encode(data)
+            stub = DEncode.encode(data).decode()
```

Comment:
this class I think better inherit `Client`

Reply:
I don't understand what you mean here?

Reply:
I mean that `DEncode` or `JEncode` encoding should be inside `Client` logic, IIUC

Reply:
The `UserProfileClient` stores `DEncode`-ed objects in the database. Maybe that should be changed but definitely not in this PR.